ddtrace/opentracer/tracer.go

```diff
@@ -47,7 +50,9 @@ func (t *opentracer) StartSpan(operationName string, options ...opentracing.Star
 	}
 	opts := []ddtrace.StartSpanOption{tracer.StartTime(sso.StartTime)}
 	for _, ref := range sso.References {
-		if v, ok := ref.ReferencedContext.(ddtrace.SpanContext); ok && ref.Type == opentracing.ChildOfRef {
+		// opentracing.ChildOfRef and opentracing.FollowsFromRef will be represented as children
+		// Datadog tracing UI does not have a concept of Follows From references
+		if v, ok := ref.ReferencedContext.(ddtrace.SpanContext); ok  {
 			opts = append(opts, tracer.ChildOf(v))
 			break // can only have one parent
```

Comment:
```
break // can only have one parent
```

maybe it would be ideal to prioritize a `ChildOfRef` if both `ChildOfRef` and `FollowsFromRef` are specified...

Reply:
True. I'm not sure if there are scenarios where a span might follow from a parent but be a child of another. If such a case exists, then we should consider it and follow this comment.